installer/Migration/V5_1_0/translation/da_DK.yaml

```diff
@@ -1,66 +1,68 @@
-    - { target: 'Indtast kommaseparerede ord', unitId: enter_comma_se, group: recruitment }
-    - { target: 'Tilføj kandidat', unitId: candidate, group: recruitment }
-    - { target: 'Nomineret', unitId: shortlisted, group: recruitment }
-    - { target: 'Interview Planlagt', unitId: interview_scheduled, group: recruitment }
-    - { target: 'Interview Bestået', unitId: interview_passed, group: recruitment }
-    - { target: 'Interview mislykkedes', unitId: interview_failed, group: recruitment }
-    - { target: 'Job Tilbydes', unitId: job_offered, group: recruitment }
-    - { target: 'Manual', unitId: manual, group: recruitment }
-    - { target: 'online', unitId: online, group: recruitment }
-    - { target: 'Contact Number', unitId: contact_number, group: recruitment }
-    - { target: 'Genoptag', unitId: resume, group: recruitment }
-    - { target: 'Tilføj kandidat', unitId: add_candidate, group: recruitment }
-    - { target: April, unitId: april, group: general }
-    - { target: August, unitId: august, group: general }
-    - { target: December, unitId: december, group: general }
-    - { target: Februar, unitId: february, group: general }
-    - { target: Fredag, unitId: friday, group: general }
-    - { target: Januar, unitId: january, group: general }
-    - { target: Juli, unitId: july, group: general }
-    - { target: Juni, unitId: june, group: general }
-    - { target: Marts, unitId: march, group: general }
-    - { target: Maj, unitId: may, group: general }
-    - { target: Mandag, unitId: monday, group: general }
-    - { target: November, unitId: november, group: general }
-    - { target: Oktober, unitId: october, group: general }
-    - { target: Lørdag, unitId: saturday, group: general }
-    - { target: September, unitId: september, group: general }
-    - { target: Søndag, unitId: sunday, group: general }
-    - { target: Torsdag, unitId: thursday, group: general }
-    - { target: Tirsdag, unitId: tuesday, group: general }
-    - { target: Onsdag, unitId: wednesday, group: general }
-    - { target: Fre, unitId: fri, group: general }
-    - { target: Man, unitId: mon, group: general }
-    - { target: Sat, unitId: sat, group: general }
-    - { target: Søn, unitId: sun, group: general }
-    - { target: Tor, unitId: thu, group: general }
-    - { target: Tir, unitId: tue, group: general }
-    - { target: Ons, unitId: wed, group: general }
-    - { target: Jan, unitId: jan, group: general }
-    - { target: Feb, unitId: feb, group: general }
-    - { target: Mar, unitId: mar, group: general }
-    - { target: Apr, unitId: apr, group: general }
-    - { target: Jun, unitId: jun, group: general }
-    - { target: Jul, unitId: jul, group: general }
-    - { target: Aug, unitId: aug, group: general }
-    - { target: Sep, unitId: sep, group: general }
-    - { target: Oct, unitId: oct, group: general }
-    - { target: Nov, unitId: nov, group: general }
-    - { target: Dec, unitId: dec, group: general }
-
```

Comment:
Don't delete other lang string please :exclamation:

Reply:
it's already in 5.0

Reply:
which one i only delete **reject** lang-string